tiledb/ml/models/_base.py

```diff
@@ -94,6 +85,58 @@ def load(self, *, timestamp: Optional[Timestamp] = None) -> Model:
     def preview(self) -> str:
         """Abstract method that previews a machine learning model."""
 
+    def _get_file_properties(
+        self,
+    ) -> Mapping[str, str]:
+        if self.Framework != "TensorBoard":
```

Comment:
This method should be overriden in `TensorBoardTileDB` instead of conditional logic in the base class.